diff --git a/node_modules/react-native-elements/src/avatar/Avatar.js b/node_modules/react-native-elements/src/avatar/Avatar.js
index 612e327..8f36ba1 100644
--- a/node_modules/react-native-elements/src/avatar/Avatar.js
+++ b/node_modules/react-native-elements/src/avatar/Avatar.js
@@ -78,7 +78,7 @@ const AvatarComponent = ({
     ));
 
   // Remove placeholder styling if we're not using image
-  const hidePlaceholder = !(source && source.uri);
+  const hidePlaceholder = false //!(source && source.uri);
 
   // Merge image container style
   const imageContainerStyle = StyleSheet.flatten([
